prow/config/tide.go

```diff
@@ -80,7 +79,7 @@ type Tide struct {
 
 	// A key/value pair of an org/repo as the key and merge method to override
 	// the default method of merge. Valid options are squash, rebase, and merge.
-	MergeType map[string]github.PullRequestMergeType `json:"merge_method,omitempty"`
+	MergeType map[string]PullRequestMergeType `json:"merge_method,omitempty"`
```

Comment:
Did you have a cycle? This seems unrelated to your changes

Reply:
Yes, there was a cycle, and those constants are used only in config package

Reply:
This is outstanding, we should not move github types out of the package. Should determine a different way to structure this to avoid the cycle

Reply:
A separate package declaring an interface that the config impl can satisfy is often the approach

Reply:
I will check that approach, and move them back to github package.